src/wallet/wallet.h

```diff
@@ -475,7 +475,7 @@ class CWalletTx : public CMerkleTx
 };
 
 
-
+using CInputCoin = std::pair<const CWalletTx*, unsigned int>;
```

Comment:
Prefer to use a typedef (I actually prefer using personally, but some devs are unfamiliar with it and you don't need it as CInputCoin is not templated)

Also, prefer a non-Hungarian name if you're introducing a new class.
See https://github.com/bitcoin/bitcoin/pull/9472#discussion_r95775684 -- Hungarians for new code discouraged.